tests/cypress/integration/admin/10-advanced-configurations.specs.js

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import adavanceConfFalseJson from "../../fixtures/api_artifacts/advanceConfigFlase.json"
```

Comment:
Shall we fix the spelling mistake here? ad**a**vanceConfFalseJson
 Better to check the usages of this and fix those as well. The file name also will have to be renamed. advanceConfig**Flase**.json

Reply:
Resolved with commit e159e6db870377cf9dc4726b102cf6292273660e